core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/definition/component/trace/componentTrace/HttpComponentHeader.java

```diff
@@ -0,0 +1,24 @@
+package io.metadew.iesi.metadata.definition.component.trace.componentTrace;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class HttpComponentHeader {
+
+    @Builder
+    public HttpComponentHeader(UUID id, HttpComponentHeaderKey httpComponentHeaderID, String name, String value) {
```

Comment:
id should be of class HttpComponentHeaderTraceKey 
HttpComponentHeaderKey should be ComponentTraceKey